nav2_navfn_planner/src/navfn.cpp

```diff
@@ -667,6 +672,10 @@ NavFn::propNavFnAstar(int cycles)
 
   // do main cycle
   for (; cycle < cycles; cycle++) {  // go for this many cycles, unless interrupted
+    if (cycle % cancel_check_interval == 0 && cancelChecker()) {
```

Comment:
Where is cancel_check_interval set and parameterized?

Reply:
I parametrized it now for other planners but it wasn't so clear how nicely do it for NavFn, so I made it constant for now.

Reply:
Where is that constant set? As far as I see, its just instantiated (so 0) but never set